services/venice-controller/src/main/java/com/linkedin/venice/controller/kafka/consumer/AdminExecutionTask.java

```diff
@@ -490,7 +490,9 @@ private void handleSetStore(UpdateStore message) {
         .setReplicationFactor(message.replicationFactor)
         .setMigrationDuplicateStore(message.migrationDuplicateStore)
         .setLatestSupersetSchemaId(message.latestSuperSetValueSchemaId)
-        .setBlobTransferEnabled(message.blobTransferEnabled);
+        .setBlobTransferEnabled(message.blobTransferEnabled)
+        .setUnusedSchemaDeletionEnabled(message.unusedSchemaDeletionEnabled)
+        .setLatestSupersetSchemaId(message.latestSuperSetValueSchemaId);
```

Comment:
Emmm..
Is there any prod impact caused by the missing statement?

Reply:
Not yet. `setLatestSupersetSchemaId` was duplicate statement. I will remove it.